### DISCUSSION TITLE
Is there an ospo guide for projects in the transitioning phase (oss to proprietary or proprietary to oss)?

### DISCUSSION THREAD

**Discussion by dineshr93** [todogroup/ospology#66] [OSPO Discussions]
Hi all, I am looking for different oss strategies or case studies for projects in the transitioning phase in either way (from proprietary to oss and from oss to proprietary. ). A guide to achieve both based on typical requirements that project or product holds. I checked  https://github.com/todogroup/outbound-oss/tree/main/content
It's for either contributing or start new . But not transitioning case .

**Comment by anajsana** [comment]
@jlprat replied:

For the first one (proprietary to OSS) you can check: https://github.com/todogroup/ospology/discussions/65 Might have nice pointers

**Comment by anajsana** [comment]
@dineshr93 Two things:

> I checked https://github.com/todogroup/outbound-oss/tree/main/content It's for either contributing or start new . But not transitioning case.

1) I'm one of the contributors to the `outbound-oss` guide and I'll be happy to bring this as a request to see if it makes sense to add this topic there. Also, if you have any ideas on how to improve the content, please open an issue there 👍 😊 

2) I think there is not a specific guide in the TODO guides. But I think there might be [room for a new TODO guide if you and more folks would like to contribute and be acknowledged ](https://todogroup.org/guides/todo-guides-contribution-guidelines/)👍 People usually gathers in the Work Day meetings to work on WIP guides and whitepapers: https://github.com/todogroup/work-day-activities
